core/src/styles/core/_form.scss

```diff
@@ -2,7 +2,67 @@
 @use '../mixins' as *;
 
 // ---- Form
+$cat-input-height: 2.5rem;
 
 .cat-error {
```

Comment:
as this is also deprecated, we can remove the _form file altogether after you have moved the code.